Services/communication_service.c

```diff
@@ -17,7 +17,25 @@
  * @date 2020-09-25
  */
 
+#include "service_response.h"
+#include "service_utilities.h"
+#include "system.h"
 #include "communication_service.h"
+#include "comms_hal.h"
+#include "uhf_hal.h"
+
+static uint8_t SID_byte = 1;
```

Comment:
Maybe we should #define this instead

Reply:
ok, I'll update the other services, too, then.

Reply:
Sounds good